cmd/constants/env.go

```diff
@@ -0,0 +1,8 @@
+// Copyright 2022 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package constants
+
+const (
+	EnvHTTPProbePort = "HTTP_PROBE_PORT"
+)
```

Comment:
do we really need this as a separate file?

Reply:
Yeah i separated them because we strictly watch the imports of the commands. So having them in another package potentially brings in a bunch of unnecessary imports that we don't want.